src/main/java/org/wikipedia/dataclient/mwapi/ImageDetails.java

```diff
@@ -0,0 +1,28 @@
+package org.wikipedia.dataclient.mwapi;
+
+import com.google.gson.annotations.SerializedName;
+
+public class ImageDetails {
+
+    @SerializedName("name")
```

Comment:
You do not need the `SerializedName` annotation if it's the same as the actual field name.